sns/src/main/scala/akka/stream/alpakka/sns/scaladsl/SnsPublishSink.scala

```diff
@@ -0,0 +1,18 @@
+package akka.stream.alpakka.sns.scaladsl
+
+import akka.Done
+import akka.stream.alpakka.sns.SnsPublishSinkStage
+import akka.stream.scaladsl.Sink
+import com.amazonaws.services.sns.AmazonSNSAsync
+
+import scala.concurrent.Future
+
+object SnsPublishSink {
+
+  /**
+   * Scala API: creates a [[SnsPublishSinkStage]] for a SNS topic using an [[AmazonSNSAsync]]
+   */
+  def apply(topicArn: String)(implicit snsClient: AmazonSNSAsync): Sink[String, Future[Done]] =
```

Comment:
I don't think the client should be implicit, especially considering the note about thread safety.